.github/workflows/tests.yml

```diff
@@ -94,6 +94,24 @@ jobs:
           threshold_warning: 50
           threshold_metric: "lines"
           comment_footer: false
+  workflow-e2e-tests:
+    name: Workflow E2E Tests
+    runs-on: ubuntu-latest
+    env:
+      PROJECT_NAME: aest-sims
+      BUILD_ID: ${{ github.event.number }}
+      BUILD_REF: ${{ github.base_ref }}
+      ZEEBE_ADDRESS: zeebe-test:26500
```

Comment:
Do we need the `ZEEBE_ADDRESS` here since it is hard-coded at the docker compose?